azext_edge/edge/providers/check/cloud_connectors.py

```diff
@@ -219,6 +195,40 @@ def process_cloud_connector(
     return check_manager.as_dict(as_list)
 
 
+def filter_connector_and_topic_map_resources(
+    connector_resource_name: str,
+    topic_map_reference_key: str,
+    all_connectors: List[Dict[str, Any]],
+    all_topic_maps: List[Dict[str, Any]],
+):
```

Comment:
nit: typing - especially for something with a complex return type like `tuple[list, list]`